documentation/api/development.md

```diff
@@ -0,0 +1,72 @@
+# Development
+
+This application is dockerized. Take a look at [Dockerfile](../../api/Dockerfile) to see how it works.
+
+A very simple [docker-compose.yml](../../docker-compose.yml) has been included to support local development and deployment.
+
+## Prerequisites
+
+**Note:** Run everything from within the `/api` folder:
+
+1. Install the version of Python specified in [pyproject.toml](../../api/pyproject.toml)
+   [pyenv](https://github.com/pyenv/pyenv#installation) is one popular option for installing Python,
```

Comment:
This link next to the previous link looks confusing. Consider adding a period in between and starting a new sentence.